Tools/MonoGame.Effect.Compiler/Effect/ShaderProfile.DirectX12.cs

```diff
@@ -80,7 +80,7 @@ internal override ShaderData CreateShader(ShaderResult shaderResult, string shad
 
                 // Compile the shader once just to get reflection info.
                 string stdout, stderr;
-                var result = ExternalTool.Run("dxc", toolArgs + "\"" + inputFile + "\"", out reflectionData, out stderr);
+                var result = Dxc.Run(toolArgs + "\"" + inputFile + "\"", out reflectionData, out stderr);
```

Comment:
@harry-cpp are we SURE this is going to work?  

Every time we've done this change we're met with months of "my content won't build" because the binary cannot be found.

Reply:
Are we also going to have to add a "Windows Kit" installer step to the build automation, or will the DXC install be picked up by (shudder) NuGet?